modules/slowzones/map/DirectionIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import type { SlowZoneDirection } from './segment';
+
+import styles from './DirectionIndicator.module.css';
+
+type Props = {
+  direction: SlowZoneDirection;
+  color: string;
+  isHorizontal: boolean;
+  size?: number;
+};
+
+const DirectionIndicator = (props: Props) => {
```

Comment:
Same thing as I stated elsewhere
```suggestion
const DirectionIndicator: React.FC<Props> = (props) => {
```